src/storageCleanupQueueOps.ts

```diff
@@ -20,6 +20,8 @@ resolver.define("storageCleanupListener", async (queueItem) => {
       console.log(`Deleting cache result for content ID ${eventPayload.contentId}...`);
       await summariserStorageUtil.deleteCacheResult(eventPayload.contentId);
     } else {
+      // 1. When will this line be executed?
```

Comment:
It is mostly about defensive programming to decouple assumptions about when items are re-cached. Currently, it would be unlikely that this line would be called - just a race condition in the same page being summarised concurrently.